src/connector/src/sink/encoder/json.rs

```diff
@@ -262,6 +284,25 @@ fn datum_to_json_object(
     Ok(value)
 }
 
+fn json_converter_with_schema<'a>(
+    object: Value,
+    name: Option<String>,
```

Comment:
Why do we need an Option here?

Reply:
Just a copy of old code, will remove it.